app.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const app = express();
+const {getTopics, catchAll} = require("./controllers/topics.controllers");
+const { handleCustomErrors, handlePsqlErrors} = require("./errors");
+
+app.use(express.json());
```

Comment:
We shouldn't have any code in here that's not being used. Is this necessary yet? Please remove it before your next commit 🙂